fabfile.py

```diff
@@ -407,6 +407,29 @@ def workload(c, upload=None, describe=None):
     
     return current_workload or None
 
+# The function will take a specified snapshot_id and delete the corresponding snapshot.
+@task
+def snapshot_delete(c, snapshot_id=None):
+    if not snapshot_id:
+        print("Please specify the snapshot(s) you wish to delete through \
+              --snapshot_id=<snapshotid,snapshotid1> separated by a ',' .")
+    else:
+        if "," in snapshot_id:
+            snapshot_ids = snapshot_id.split(",")
+            for snapshot_id in snapshot_ids:
+                result = c.run("aws ec2 delete-snapshot \
+                --snapshot-id {snapshot_id}")
+                if result.stderr:
+                    raise Exit(result.stderr)
```

Comment:
Should we print the error but still try to delete the other snapshots?

Reply:
I think it is risky to continue deleting when one is wrong.

Reply:
Yes, that's fair.